recipes/meta/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "meta" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
```

Comment:
What's the name that other package managers use?

Reply:
Hi [isuruf](https://github.com/isuruf)
I previously tried to publish this package on PyPI (under the name meta-neuro) but faced challenges due to the mix of Python and C++ code. I updated the package, and I’d like to use the name “meta” for this Conda package, where “meta” stands for Medial Tractography Analysis (MeTA). Thank you!

Reply:
Please use `meta-neuro`. The name `meta` is too generic.

Reply:
Okay, I updated the package name to `meta-neuro`.